src/Calendar/Calendar.js

```diff
@@ -69,40 +59,54 @@ class Calendar extends Component {
         });
     }
 
-    displayBetweenRange = (day) => {
-        return this.props.enableRangeSelection && this.isDateBetween(day, this.state.arrSelectedDates, this.props.enableRangeSelection);
-    }
-
-    displaySelectedRangeLast = (day) => {
-        return this.props.enableRangeSelection && (typeof this.state.arrSelectedDates[1] !== 'undefined' && this.state.arrSelectedDates[1].getTime() === day.getTime());
+    isEnabledDate = (day) => {
+        const {
+            disableWeekends,
+            disableAfterDate,
+            disableBeforeDate,
+            blockedDates,
+            disableWeekday,
+            disablePastDates,
+            disableFutureDates,
+            disabledDates
+        } = this.props;
```

Comment:
These props now accept `Date` objects again instead of moment objects so consumers don't need to worry about importing moment. The Dates are just wrapped in moment inside. This fixes the entire moment objects' props being listed out on the docs site, but the API might actually be better handled this way in the first place.